src/testoptions.c

```diff
@@ -96,18 +96,6 @@ void findCwndPeaks(tcp_stat_agent* agent, CwndPeaks* peaks,
   prevCWNDval = CurCwnd;
 }
 
-/**
- * Print the appropriate message when the SIGALRM is caught.
- * @param signo The signal number (shuld be SIGALRM)
- */
-
```

Comment:
I have added comments throughout the server code in the codebase all referencing `cleanup()` and added an explanation of the behavior in the `SIGALRM` case in that function.